trustme/_cli.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+import trustme
+import sys
+
+# Python 2/3 annoyingness
+try:
+    unicode
+except NameError:  # pragma: no cover
+    unicode = str
```

Comment:
For what it's worth, I think we're happy to remove Python 2 support in another pull request.